.github/workflows/test.yml

```diff
@@ -69,9 +69,10 @@ jobs:
           - disk
           - member
           - partition,kill,stop
-          - partition,kill,disk
+          - partition,kill
           - partition,kill,member
-          - partition,pause,disk
+          - partition,disk
+          - pause,disk
```

Comment:
Yes.
In my branch I've been experimenting with:
```yaml
matrix:
  workload: [append, set, bank]
  nodes:    ['', stop, kill, member]
  network:  ['', partition]
  process:  ['', pause]
  storage:  ['', disk]
  exclude:
    # TODO: member doesn't package well with all `nemesis.combined`, so selective use
    - nodes: member
      process: pause
    - ...
    # filling up the disk when nodes come and go can ultimately create an environment that isn't capable of supporting a cluster
    - storage: disk
      nodes: stop
  include:
    # augment set with no nemeses with a time-limit, otherwise Java OOM
    - workload: set
      ...
      time-limit: 180
```

To explore all viable combinations.
Also want to add expected pass and expected fail configuration.

Reply:
The matrix config in this PR contains the minimum adjustments needed to accommodate the increased efficiency and behavior of the updated disk nemesis.

I was thinking that the comprehensive matrix, expected fail/pass, and a custom dev configuration would be it's own PR.
e.g. a larger configuration change the team can evaluate for fit with their usage/workflow.

P.S. There's one more nemesis review PR coming, for start/stop/kill.
And then focus on configuration, usability, experience.